tests/unit/test_rust_backend.py

```
@@ -24,7 +22,10 @@
 
 # Import any custom exceptions
 try:
-    from git.markdown_lab.core.rust_backend import RustBackendError, RustCompilationError
+    from git.markdown_lab.core.rust_backend import (
+        RustBackendError,
+        RustCompilationError,
+    )
```

Pylint (3.3.7)

[convention] Imports from package git are not grouped

(C0412)